tools/profiles/release.json

```diff
@@ -14,6 +14,13 @@
                "-Wl,--wrap,_calloc_r", "-Wl,--wrap,exit", "-Wl,--wrap,atexit",
                "-Wl,-n"]
     },
+    "ARMC6": {
+        "common": ["-c", "--target=arm-arm-none-eabi", "-mthumb", "-Oz", "-DNDEBUG", "-flto"],
```

Comment:
isn't this duplicate `flto` as it is also defined in the `ld` ?

Curious about choosing - `Oz` for the release?